kmod/patch/livepatch-patch-hook.c

```diff
@@ -248,11 +258,9 @@ static int __init patch_init(void)
 		goto out;
 	lpatch->mod = THIS_MODULE;
 	lpatch->objs = lobjects;
-#if __powerpc__
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 12, 0))
+#if defined(__powerpc__) && defined(HAVE_IMMEDIATE)
```

Comment:
BTW there are a bunch of more `#ifdef __powerpc__` instances in create-diff-object.c and kpatch-elf.c.

Reply:
The ones in create-diff-object.c and kpatch-elf.c aren't specific to kernel versions so I think they're fine as they are.